apis/io-engine/protobuf/v1/common.proto

```diff
@@ -7,3 +7,32 @@ enum ShareProtocol {
   NVMF = 1; // NVMe over Fabrics (TCP)
   ISCSI = 2; // iSCSI
 }
+
+// Common IO stats struct for all resource types.
+message IoStats {
```

Comment:
let's expose the max counters too so we can pick it up from io-engine client at least; export can just ignore those

Reply:
Sure